west.yml

```diff
@@ -116,7 +116,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: bebb88e59655efb172b295a77d5a1eb62dc4552e
+      revision: pull/883/head
```

Comment:
Please update SHA: `51f60b21b708368be1437c613129a54f4682d593`

Reply:
@rlubos  I rebased and updated the sha